sw/device/silicon_creator/manuf/base/provisioning_inputs.bzl

```diff
@@ -25,7 +25,8 @@ EARLGREY_SKUS = {
         "dice_libs": ["//sw/device/silicon_creator/lib/cert:dice"],
         "host_ext_libs": ["@provisioning_exts//:default_ft_ext_lib"],
         "device_ext_libs": ["@provisioning_exts//:default_perso_fw_ext"],
-        "transport_image": "//sw/device/silicon_creator/rom_ext/sival:rom_ext_fake_prod_signed_slot_b",
+        "rom_ext": "//sw/device/silicon_creator/rom_ext/sival:rom_ext_fake_prod_signed_slot_b",
+        "owner_fw": "//sw/device/silicon_owner/bare_metal:bare_metal_slot_b",
```

Comment:
where is `bare_metal_slot_b` defined? 

Maybe adding some "theory of operation" text to the patch description would help

Reply:
It is defined here: https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_owner/bare_metal/BUILD;l=82?q=bare_metal&ss=opentitan%2Fopentitan (it is just a placeholder for sival SKU). You can put whatever you want for owner_fw in your SKU.

Updated the patch description; let me know if you were looking for something else.

Reply:
somehow it did not make it to patch description, and I am not entirely clear now the SKU can specify this input...